### PR TITLE
Repack

### PR DESCRIPTION
**Pull request by Elad-Laufer**

### CODE REVIEW

File: vips/govips.go [hunk 1]
```diff
@@ -20,15 +15,6 @@ const (
 	defaultMaxCacheSize     = 500
 )
 
-// VipsVersion if the primary version of libvips
-const VipsVersion = string(C.VIPS_VERSION)
```
Comment (alon-ne) [inline vips/govips.go]: Why remove the vips version constants? It's good that they're there

File: vips/type.go [hunk 2]
```diff
@@ -43,9 +43,6 @@ type Color struct {
 
 const DefaultFont = "sans 10"
 
-// ColorBlack is shorthand for black RGB
-var ColorBlack = Color{0, 0, 0}
```
Comment (alon-ne) [inline vips/type.go]: Looks like a useful constant for users of this package

File: vips/govips.go (hunk 1 above)
Comment (Elad-Laufer) [reply vips/govips.go]: fine, also added patch (micro) while at it

File: vips/type.go (hunk 2 above)
Comment (Elad-Laufer) [reply vips/type.go]: fine

**Pull request merged**